corehq/apps/hqwebapp/templates/hqwebapp/partials/form_list_form.html

```diff
@@ -25,7 +25,7 @@
         </thead>
 
         <tbody data-bind="foreach: rows">
-            <tr>
+            <tr data-bind="css: { error: Object.keys(errors).length !== 0 }">
```

Comment:
Good to know, thanks!  I was shocked to find out that JS doesn't seem to have any good way of doing `my_hashmap.isEmpty;`, `my_hashmap === {}`, or `my_hashmap.length === 0;`